WooCommerce/Classes/Analytics/WooAnalytics.swift

```diff
@@ -122,16 +175,31 @@ public extension WooAnalytics {
             return
         }
 
-        let properties = combinedProperties(from: error, with: passedProperties)
```

Comment:
The error properties logic has been moved to the `WooAnalytics.track(_ eventName:properties:error:)` since the error properties are independent of `WooAnalyticsStat`